openmc/mgxs/__init__.py

```diff
@@ -123,6 +165,80 @@
+    1.33297e+01, 1.35460e+01, 1.40496e+01, 1.42505e+01, 1.44702e+01,
+    1.45952e+01, 1.47301e+01, 1.48662e+01, 1.57792e+01, 1.60498e+01,
+    1.65501e+01, 1.68305e+01, 1.74457e+01, 1.75648e+01, 1.77590e+01,
+    1.79591e+01, 1.90848e+01, 1.91997e+01, 1.93927e+01, 1.95974e+01,
+    2.00734e+01, 2.02751e+01, 2.04175e+01, 2.05199e+01, 2.06021e+01,
+    2.06847e+01, 2.07676e+01, 2.09763e+01, 2.10604e+01, 2.11448e+01,
+    2.12296e+01, 2.13360e+01, 2.14859e+01, 2.17018e+01, 2.20011e+01,
+    2.21557e+01, 2.23788e+01, 2.25356e+01, 2.46578e+01, 2.78852e+01,
+    3.16930e+01, 3.30855e+01, 3.45392e+01, 3.56980e+01, 3.60568e+01,
+    3.64191e+01, 3.68588e+01, 3.73038e+01, 3.77919e+01, 3.87874e+01,
+    3.97295e+01, 4.12270e+01, 4.21441e+01, 4.31246e+01, 4.41721e+01,
+    4.52904e+01, 4.62053e+01, 4.75173e+01, 4.92591e+01, 5.17847e+01,
+    5.29895e+01, 5.40600e+01, 5.70595e+01, 5.99250e+01, 6.23083e+01,
+    6.36306e+01, 6.45923e+01, 6.50460e+01, 6.55029e+01, 6.58312e+01,
+    6.61612e+01, 6.64929e+01, 6.68261e+01, 6.90682e+01, 7.18869e+01,
+    7.35595e+01, 7.63322e+01, 7.93679e+01, 8.39393e+01, 8.87741e+01,
+    9.33256e+01, 9.73287e+01, 1.00594e+02, 1.01098e+02, 1.01605e+02,
+    1.02115e+02, 1.03038e+02, 1.05646e+02, 1.10288e+02, 1.12854e+02,
+    1.15480e+02, 1.16524e+02, 1.17577e+02, 1.20554e+02, 1.26229e+02,
+    1.32701e+02, 1.39504e+02, 1.46657e+02, 1.54176e+02, 1.63056e+02,
+    1.67519e+02, 1.75229e+02, 1.83295e+02, 1.84952e+02, 1.86251e+02,
+    1.87559e+02, 1.88877e+02, 1.90204e+02, 1.93078e+02, 1.95996e+02,
+    2.00958e+02, 2.12108e+02, 2.24325e+02, 2.35590e+02, 2.41796e+02,
+    2.56748e+02, 2.68297e+02, 2.76468e+02, 2.84888e+02, 2.88327e+02,
+    2.95922e+02, 3.19928e+02, 3.35323e+02, 3.53575e+02, 3.71703e+02,
+    3.90760e+02, 4.19094e+02, 4.53999e+02, 5.01746e+02, 5.39204e+02,
+    5.77146e+02, 5.92941e+02, 6.00099e+02, 6.12834e+02, 6.46837e+02,
+    6.77287e+02, 7.48517e+02, 8.32218e+02, 9.09681e+02, 9.82494e+02,
+    1.06432e+03, 1.13467e+03, 1.34358e+03, 1.58620e+03, 1.81183e+03,
+    2.08410e+03, 2.39729e+03, 2.70024e+03, 2.99618e+03, 3.48107e+03,
+    4.09735e+03, 5.00451e+03, 6.11252e+03, 7.46585e+03, 9.11881e+03,
+    1.11377e+04, 1.36037e+04, 1.48997e+04, 1.62005e+04, 1.85847e+04,
+    2.26994e+04, 2.49991e+04, 2.61001e+04, 2.73944e+04, 2.92810e+04,
+    3.34596e+04, 3.69786e+04, 4.08677e+04, 4.99159e+04, 5.51656e+04,
+    6.73794e+04, 8.22974e+04, 9.46645e+04, 1.15624e+05, 1.22773e+05,
+    1.40000e+05, 1.64999e+05, 1.95008e+05, 2.30014e+05, 2.67826e+05,
+    3.20646e+05, 3.83884e+05, 4.12501e+05, 4.56021e+05, 4.94002e+05,
+    5.78443e+05, 7.06511e+05, 8.60006e+05, 9.51119e+05, 1.05115e+06,
+    1.16205e+06, 1.28696e+06, 1.33694e+06, 1.40577e+06, 1.63654e+06,
+    1.90139e+06, 2.23130e+06, 2.72531e+06, 3.32871e+06, 4.06569e+06,
+    4.96585e+06, 6.06530e+06, 6.70319e+06, 7.40817e+06, 8.18730e+06,
+    9.04836e+06, 9.99999e+06, 1.16183e+07, 1.38403e+07, 1.49182e+07,
+    1.96403e+07])
```

Comment:
This array has 361 values, which means there would be 360 groups, not 361. Is there a value missing?

Reply:
My bad. The lowest value was not included where I found this. 
I'm placing a 0eV bound there